_schema.graphql

```diff
@@ -8076,7 +8082,7 @@ input MoneyInput {
 }
 
 # A money object with amount in minor currency and a currency code, or an error object
-union MoneyOrErrorUnion = Error | Money
+union MoneyOrErrorUnion = Error
```

Comment:
Everything’s free! 💸

Reply:
Perhaps there was an internal discussion about this, but I don’t quite understand this change, it should be a union of an error and a money object, right?

Reply:
@alloy fair question! Yes, this is a `Error | Money` union in Gravity's GQL types/schema. In this case, the `Money` portion was removed automatically during schema stitching after we specifically stopped overwriting MP's `Money` type with Gravity's `Money` type: https://github.com/artsy/metaphysics/blob/83d8dc976142acaaffac9b65a550bee057a0cff2/src/lib/stitching/gravity/schema.ts#L24-L33

We couldn't quickly figure out how to remove the `MoneyOrErrorUnion` completely so punted on it for now - didn't seem like it was worth slowing down this PR since we don't expect `MoneyOrErrorUnion` to actually be used in MP. But we're also pretty new to MP! Open to feedback